src/components/Collapse/index.tsx

```diff
@@ -7,14 +7,29 @@ import styled from 'styled-components'
 
 const Wrapper = styled.div``
 
+const HeaderWrapper = styled.div``
+
+const Title = styled.div``
```

Comment:
actually, here I prefer it to be `null` as it can be a simple text or a ReactNode.

Also, if it's `null` it won't generate a new node in the DOM and I don't need to make further checks to prevent that, as in the `''` case.

Reply:
I think you were talking about `description = null` thing

Reply:
hehehe, yes :grimacing: